common/testParameters.js

```diff
@@ -1408,6 +1408,111 @@ export const acceptedCharInMask = _.filter((t) => t, [{
 //####################################################################################
 //####################################################################################
 
+// export const someCharsRejected = _.filter((t) => false, [{
+export const someCharsRejected = _.filter((t) => t, [{
+// export const someCharsRejected = _.filter((t) => t.only, [{
+  line: getLineNumber(),
+
+  // placeholder _
+  mask: [/\d/],
+  previousConformedValue: '',
+  rawValue: '1',
+  currentCaretPosition: 1,
+
+  someCharsRejected: false
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder _
+  mask: [/\d/],
+  previousConformedValue: '',
+  rawValue: 'a',
+  currentCaretPosition: 1,
+
+  someCharsRejected: true
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder $
+  mask: ['$'],
+  previousConformedValue: '',
+  rawValue: 'aaa',
+  currentCaretPosition: 3,
+
+  someCharsRejected: true
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder $
+  mask: ['$'],
+  previousConformedValue: '',
+  rawValue: 'a',
+  currentCaretPosition: 1,
+
+  someCharsRejected: true
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder $_
+  mask: ['$', /\d/],
+  previousConformedValue: '',
+  rawValue: 'a',
+  currentCaretPosition: 1,
+
+  someCharsRejected: true
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder $
+  mask: ['$'],
+  previousConformedValue: '',
+  rawValue: '',
+  currentCaretPosition: 1,
+
+  someCharsRejected: false
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder
+  mask: [],
+  previousConformedValue: '',
+  rawValue: '',
+  currentCaretPosition: 0,
+
+  someCharsRejected: false
+
+  // only: true
+}, {
+  line: getLineNumber(),
+
+  // placeholder $
+  mask: ['$'],
+  previousConformedValue: '',
+  rawValue: '$',
+  currentCaretPosition: 0,
+
+  someCharsRejected: false
```

Comment:
I'm unsure what the correct value for `someCharsRejected` should be in this test.

In the case of the `createNumberMask`, if a user typed the currency sumbol, (`$`), should it be flagged as a rejected character... also consider the case where the input was `$1|` and the user presses backspace to delete the `1 leaving the value as `$`.